src/main/java/com/thoughtworks/gauge/datastore/SuiteDataStore.java

```diff
@@ -38,12 +38,16 @@ public static synchronized Object remove(Object key) {
      */
     public static synchronized Object get(Object key) {
         if (key != null) {
-            return map.get().get(key);
+            return getMap().get(key);
         }
         return null;
     }
 
+    public static synchronized ConcurrentHashMap<Object, Object> getMap() {
```

Comment:
The DataStore deliberately avoided returning a Map to hide the internal storage implementation. Exposing this would make changing the implementation of the DataStore harder in the future. 

Is there a good reason why the getMap methods needs to be public?

Reply:
I was just looking to get a list of entries. I can change these to return an entrySet instead?

Reply:
I've now updated to return entrySet only which we also did previously in the old DataStore.